### PR TITLE
avoid writing configuration file which can create permission issues in container environment

### PR DESCRIPTION
**Pull request by TejasMorbagal**
On deeper introspection, I realize calling copernicusmarine.login() is not necessary as I could pass username and password directly to copernicusmarine.describe and copernicusmarine.open_dataset().

This solves the issue #34

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## Changes in 0.1.4
+
+- Changed initialization of cmems to not call copernicusmarine.login that writes a 
+  configuration_file. (#32)
```
Comment (thomasstorm) [inline CHANGES.md:4]:
```suggestion
  configuration file. (#32)

```

**Review by thomasstorm — APPROVED** [review]
Looks good, things are made much easier!

**Review by forman — APPROVED** [review]
Ok, but you should update copyright notices asap:

`# Copyright (c) 2022 by the xcube...` -->  `# Copyright (c) 2022-2024 by the xcube...`

**Review by pont-us — APPROVED** [review]
Looks good, approving (though I only eyeballed the notebook and didn't try running it locally).

I don't see any changes to the actual calls to the backend functions in `cmems.py`, but as far as I can tell `open_dataset` was passing the credentials explicitly already, and `describe` doesn't need them -- is that right?

**Comment by TejasMorbagal** [comment]
> > Looks good, approving (though I only eyeballed the notebook and didn't try running it locally).
> > I don't see any changes to the actual calls to the backend functions in `cmems.py`, but as far as I can tell `open_dataset` was passing the credentials explicitly already, and `describe` doesn't need them -- is that right?
> 
> Yes, that's right! The creation of configuration_file in login function gave the option of skipping the actual login for the subsequent calls. This feature is not useful for us.

**Pull request merged**